### DISCUSSION TITLE
border color differs between input control and other elements

### DISCUSSION THREAD

**Discussion by yankeeinlondon** [tailwindlabs/tailwindcss#5875] [Help]
**What version of Tailwind CSS are you using?**

v2.0.4

**What build tool (or framework if it abstracts the build tool) are you using?**

available to demonstrate in Playground

**What version of Node.js are you using?**

N/A

**What browser are you using?**

Chrome and Firefox

**What operating system are you using?**

macOS

**Reproduction URL**

[playground](https://play.tailwindcss.com/MTdWGfScSj?layout=horizontal)

**Describe your issue**

I am using the Forms plugin and while this gives me a default border color, when I try to set it manually it both lets me and unfortunately produces a different color than what it should. I say this based on comparison. If you look at the playground example you'll see what I mean. Both set `border-gray-300/100` as a color:

<img width="733" alt="2021-10-22_13-22-26" src="https://user-images.githubusercontent.com/980757/138518390-3d048a83-173b-4d26-87c5-a600601b654c.png">

<img width="782" alt="2021-10-22_13-24-22" src="https://user-images.githubusercontent.com/980757/138518520-9fcf8461-bc58-465e-ae8c-bca38440e501.png">

When you compare the "computed style" for both elements they report the same RGB values but they are very visibly different in color.

<img width="338" alt="2021-10-22_13-26-47" src="https://user-images.githubusercontent.com/980757/138519902-3f193b96-9c4d-4b1a-a98c-88dc7dc21c59.png">

**Comment by yankeeinlondon** [comment]
Note: for reasons that escape me if you use one level less of luminosity for the border of a the non-control, this seems to match: [EXAMPLE](https://play.tailwindcss.com/53fRgxs3Zk?layout=horizontal)

**Comment by adamwathan** [comment]
Hey! The problem here is you have `opacity-50` set on the input, which reduces the opacity of the input to 50% (including its border) and makes the border appear lighter.

https://play.tailwindcss.com/pekk7aB1pb?layout=horizontal

**Reply by yankeeinlondon** [reply]
I do? Wow sorry to waste your time on this. I'm not sure how that snuck in there but it explains everything.
